package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@docknetwork/sdk",
-  "version": "7.2.1",
+  "version": "7.2.2",
```

Comment:
this PR contains breaking changes, eg:
`DockDidMethodKey` -> `DidMethodKey`

so the version should be bumped up higher